app/components/SelectedAlertShortcutLink.tsx

```diff
@@ -0,0 +1,52 @@
+import { Link } from '@remix-run/react'
+
+import {
+  JsonNoticeTypes,
+  NoticeTypes,
+} from './NoticeTypeCheckboxes/NoticeTypeCheckboxes'
+import { WithCredentials, useEmail } from '~/root'
+
+export default function SelectedAlertShortcutLink({
+  alertKey,
+  format,
+  otherAlerts,
+  destination,
+}: {
+  alertKey: string
+  format: 'json' | 'text'
+  destination: 'quickstart' | 'email'
+  otherAlerts?: string[] // Alerts specifically under the 'Other' tab, since they can't be generically selected as all
+}) {
+  const userCredentials = WithCredentials()
```

Comment:
Please don't pre-select a client ID for the user.

Reply:
This is necessary, at least if the point of this component is to be a shortcut to the prefilled alerts page. Otherwise this may as well just be a link to the streaming quickstart